### PR TITLE
Add files

### PR DESCRIPTION
**Pull request by Devesh123987**
waiting for the code review so that the branch can be merged into the main branch.

### CODE REVIEW

File: .github/workflows/linters.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
+name: Linters
```
Comment (Devesh123987) [inline .github/workflows/linters.yml:1]: There is some problem with the linters file kindly check the same.

**Review by joel-kalema — COMMENTED** [review]
Hi @Devesh123987 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request 

- [ ] You have submitted a project that doesn't have any inline comment. Kindly follow the list from the article ["How to give a code review"](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md).


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @joel-kalema in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+  select {
+    border-radius: 5px;
+  }
+
+  textarea {
+    min-height: 6.33em;
+    resize: vertical;
+  }
+
+  input[type="submit"] {
```
Comment (Devesh123987) [inline styles.css:64]: A padding can be added line no 64 .
I suggested you this change as it will certainly enhance the user experience.

**Comment by Devesh123987** [comment]
Thanks for the feedback,

As suggested by you I haven't used inline comment.

But I followed the instructions. I hovered over the code which needs to be
commented on and then clicked on the blue icon.

Regards
Devesh Srivastava

File: index.html [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Indian History Survey Form </title>
+    <link rel="stylesheet" href="styles.css">
+  </head>
+
+  <body>
+    <h1 id="title">India History Survey Form</h1>
+   <p id="description" class="established"></p>Thank you for taking the time to help us improve the platform</p>
```
Comment (Devesh123987) [inline index.html:11]: Please align this paragraph. Kindly center the paragraph on the page using the CSS style sheet.
Line no: 11
I suggested you this change as the paragraph is not in sync with the page layout.

File: styles.css [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+  }
+
+  p {
+    margin-top: 0;
+    margin-bottom: 2rem;
+  }
+
+  .established {
+    font-style: italic;
+  }
```
Comment (Devesh123987) [inline styles.css:22]: Kindly position the same using css property for line 20 class establishment you can use css property to center the paragraph.

**Comment by didierganthier** [comment]
Hi @Devesh123987 ,

While you put in a lot of hard work in this project, unfortunately, I cannot proceed with the review of your code.



## Invalid Code Review Request 🚫

- This project is about reviewing your own code, you did not submit a review at all. You have to review the code using the GitHub Review features and make sure to add inline comments. Please pay attention to this:
   - ["How to give a code review"](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md).

Your Code Review Request will be marked as invalid in your Dashboard, but don't worry you can always try again! 💪

Cheers and Happy coding!👏👏👏

If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread and don't forget to tag me so I'll receive the notification.

**Please note**
- To avoid confusion and delays, please do not open a new pull request for re-reviews. Use the same pull request submitted for the first review, unless otherwise instructed.

--------

_Invalid Code Review Request **does not count** towards the code reviews limit._

**Review by Devesh123987 — COMMENTED** [review]
Hey ! Grate work 
While you put in a lot of hard work in this project, I would suggest you to Style the paragraph with Id description so that it is aligned center, believe me it would much better.

Readme file can be more descriptive:
It will be really nice if the Readme template is more descriptive and contains some more details about the project.

**Review by Strangeal — CHANGES_REQUESTED** [review]
Hi @Devesh123987, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Alt Text
## Highlights
Used inline comment ✅

## Required Changes ♻️
- [x] Kindly consider phrasing your first two comments politely, as suggestions instead of orders. This will make the student more comfortable to make the changes required and also ask questions if they don't understand the requested changes.

- [ ] Please consider including an explanation, a "why", for each requested change so the student knows why that change is important and can also learn from it and not repeat the same mistake next time.
- [ ] Kindly make sure you use the inline comment appropriately on the first and second comments. When requesting a change please highlight exactly where you think the problem might be coming from `(the specific line of code)` so the student knows where to look. Just like you did in your third and fourth comments.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .github/workflows/linters.yml [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+      - name: Webhint Report
+        run: npx hint .
+  stylelint:
+    name: Stylelint
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "18.x"
```
Comment (Devesh123987) [inline .github/workflows/linters.yml:42]: Kindly check for problems with the linters.
Why: As you may know, it is a mandatory requirement.
Line no: 42

**Review by Devesh123987 — COMMENTED** [review]
Hey ! Grate work
While you put in a lot of hard work in this project, I would suggest you to Style the paragraph with Id description so that it is aligned center, believe me it would much better.

Readme file can be more descriptive:
It will be really nice if the Readme template is more descriptive and contains some more details about the project.
style.css: 
It will be a boost in the over all user experience if suggested changes are implemented.

If you are still facing any issues feel free to contact me @Devesh123987  

Happy coding!

**Review by LionRouge1 — APPROVED** [review]
Hi @Devesh123987, 

# STATUS: APPROVED! 🥇 💯 
Great work on making the changes 👏 
You've done well implementing the requested changes

done

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**